package.json

```diff
@@ -4,6 +4,7 @@
   "description": "GitHub OAuth toolset for Node.js",
   "scripts": {
     "build": "pika build",
+    "build:cloudflare": "esbuild src/middleware/cloudflare/index.ts --bundle --format=esm --outfile=dist/worker.mjs",
```

Comment:
This isn't needed. Pika automatically includes the files in `src/**/*`

Reply:
Cloudflare worker accept a single JS file. Please let me know if I need to use Pika to bundle the worker script. How should the pipeline be structured? Sorry, I’m new to Pika.

Reply:
Bundling should be left to end-users. We only compile the source down to plain JavaScript, which Pika handles.